Extension/lib/content-script/adguard-content.js

```diff
@@ -17,8 +17,5 @@
 
 /**
  * Global object for content scripts
- * @type {{}}
  */
-var adguardContent = (function () { // jshint ignore:line
```

Comment:
Because `adguardContent` variable is just like a marker for common-script.js
```
...
})(typeof adguard !== 'undefined' ? adguard : adguardContent);
```

Messaging API implementation (`runtimeImpl.onMessage|sendMessage`) is exposed into passed variable, and then used in the content-script.js